tesseract/tesseract_planning/tesseract_time_parameterization/include/tesseract_time_parameterization/iterative_spline_parameterization.h

```diff
@@ -210,15 +210,17 @@ class IterativeSplineParameterization
    * @param max_acceleration_scaling_factor The max acceleration scaling factor. Size should be trajectory.size()
    * @return True if successful, otherwise false
    */
-  bool compute(const std::vector<std::reference_wrapper<Instruction>>& trajectory,
+  bool compute(std::vector<std::reference_wrapper<Instruction>>& trajectory,
                const Eigen::Ref<const Eigen::VectorXd>& max_velocity,
                const Eigen::Ref<const Eigen::VectorXd>& max_acceleration,
                const Eigen::Ref<const Eigen::VectorXd>& max_velocity_scaling_factors,
                const Eigen::Ref<const Eigen::VectorXd>& max_acceleration_scaling_factors) const;
 
+#ifndef SWIG
 private:
   bool add_points_;  /// @brief If true, add two points to trajectory (first and last segments).
                      /// If false, move the 2nd and 2nd-last points.
+#endif               // SWIG
```

Comment:
Does swig not automatically ignore private and protected scoped items?

Reply:
SWIG was failing parsing that line. I think it may be due to the doxygen comment being malformed. I'll trying fixing the comment.

Reply:
I fixed the comment and removed the #ifdef SWIG